File: JAVA/scanner.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // 정수 입력 받기
        System.out.print("정수를 입력하세요: ");
        int intValue = scanner.nextInt();
        System.out.println("입력한 정수: " + intValue);

        // 실수 입력 받기
        System.out.print("실수를 입력하세요: ");
        double doubleValue = scanner.nextDouble();
        System.out.println("입력한 실수: " + doubleValue);

        // 문자열 입력 받기
        System.out.print("문자열을 입력하세요: ");
        String stringValue = scanner.next();
        System.out.println("입력한 문자열: " + stringValue);

        // 입력 스트림 닫기
        scanner.close();
    }
}

File: C++/scanner.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    // 정수 입력 받기
    cout << "정수를 입력하세요: ";
    int intValue;
    cin >> intValue;
    cout << "입력한 정수: " << intValue << endl;

    // 실수 입력 받기
    cout << "실수를 입력하세요: ";
    double doubleValue;
    cin >> doubleValue;
    cout << "입력한 실수: " << doubleValue << endl;

    // 개행문자(Enter) 처리
    cin.ignore();

    // 문자열 입력 받기
    cout << "문자열을 입력하세요: ";
    string stringValue;
    getline(cin, stringValue);
    cout << "입력한 문자열: " << stringValue << endl;

    return 0;
}
